app/components/container/container.html

```diff
@@ -54,15 +54,20 @@
             <tr>
               <td>Status</td>
               <td>
-                <i ng-class="{true: 'fa fa-heartbeat space-right green-icon', false: 'fa fa-heartbeat space-right red-icon'}[container.State.Running]"></i>
-                {{ container.State|getstatetext }} since {{ activityTime }}<span ng-if="!container.State.Running"> with exit code {{ container.State.ExitCode }}</span>
+                <i ng-class="fa fa-heartbeat space-right green-icon" ng-if="container.State.Running"></i>
```

Comment:
Use `class` instead of `ng-class` here. Otherwise the icons are not displayed and an error is raised in the Javascript console.